### PR TITLE
Add Azure Security Group

### PR DESCRIPTION
**Pull request by moelsayed**

### CODE REVIEW

**Comment by moelsayed** [comment]
https://github.com/rancher/rancher/issues/8721

**Comment by alena1108** [comment]
@moelsayed I believe you also need to modify config.json file with the new version

**Comment by alena1108** [comment]
LGTM

**Comment by deniseschannon** [comment]
Can we add some updates in the etcd v2 readme about the changes that were made to the template? @moelsayed

**Comment by moelsayed** [comment]
@deniseschannon sure, working on it.

File: infra-templates/k8s/33/README.md [hunk 1]
```diff
@@ -32,3 +33,16 @@ Audit logs can be enabled through a simple option when deploying kubernetes, it
...
+
+It's possible to use Rancher-Machine-provisioned hosts or custom hosts created on Azure. Only hosts in the same resource group are currently supported. Also, for Azure Load Balancer to work, hosts that are expected to be in the Load balancer pool should be in the configured `Security Group`.
+
+The following parameters are required to use the Azure Cloud Provider:
+* **Azure Cloud Environment:** Select one of the available Azure Cloud Environments.
+* **Azure Tenant ID**: In your Azure Portal, go to `Azure Active Directory` and select `properties`. Your `Directory ID` is your `Tenant ID`. Also, you can run the command `az account show` in your Azure Shell to get the same information.
+* **Azure Client ID**: You can use the the procedure [here](http://rancher.com/docs/rancher/latest/en/hosts/azure/#app-registration) to create an App Registration.
+* **Azure Client Secret:** Created as part to the App Registration procedure.
+* **Azure Security Group:** Custom Azure Security Group needed to allow Azure Load Balancers to work. If you provision hosts using Rancher Machine Azure driver, you will need to edit them manually to assign them to this Security Group.
```
Comment (alena1108) [inline infra-templates/k8s/33/README.md:48]: @moelsayed so docker-machine driver doesn't support passing security group when provision the host?

Comment (moelsayed) [reply infra-templates/k8s/33/README.md:48]: @alena1108 No, it doesn't. they specifically [mention](https://docs.docker.com/machine/drivers/azure/#notes):
```
Once the machine is created, you can modify Network Security Group rules and open ports of the machine from the Azure Portal.```

**Comment by alena1108** [comment]
LGTM

**Pull request merged**